packages/editor-ui/src/utils/apiUtils.ts

```diff
@@ -177,10 +177,11 @@ export async function patch(
  */
 export function unflattenExecutionData(
 	fullExecutionData: IExecutionFlattedResponse,
-): Omit<IExecutionResponse, 'status'> {
+): IExecutionResponse {
 	// Unflatten the data
-	const returnData: Omit<IExecutionResponse, 'status'> = {
+	const returnData: IExecutionResponse = {
 		...fullExecutionData,
+		status: 'unknown',
```

Comment:
could this be problematic, showing wrong execution state to users? Could we solve this differently?

Reply:
Good point. Introduced a new type without `status`.